lib/diagnostic.rb

```diff
@@ -5,36 +5,37 @@ class Diagnostic
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
-    # Your code goes here
+    return list.max_by(&:length)
```

Comment:
You'll need the spaceship operator since it's a comparison

`list.max { |a , b| a.length <=> b.length }`